src/components/elements/EventTopicField.jsx

```diff
@@ -7,7 +7,7 @@ function EventTopicField () {
   return (
     <div>
       <label>Topics</label>
-      <Field name="topic" component="select" className={formStyleClasses.select}>
+      <Field name="topic" component="select" className={`${formStyleClasses.select}  dark:bg-[#151A35]`}>
```

Comment:
@Rohanfizz, I think additional class should be added to the css class object instead of being hardcoded here. Could handle that as a follow up?

Reply:
Sure, should I do it that way?

Reply:
Done✅